### ISSUE TITLE
How to set `monitor` in `keras.callbacks` according to my loss?

### ISSUE THREAD

**Issue by igo312** [keras-team/keras#13580]
I'm using `keras.callbacks.ModelCheckpoint` to save the result. Cause I get three outputs, I worry about there may be error feedback if I set `monitor = 'val_loss'`.
I know I can set `monitor` as 'val_loss', 'accuracy' and so on, but how to set `monitor` according to my ouput?
And my output is following 
```python
[<tf.Tensor 'a/Softmax:0' shape=(？, 2) dtype=float32>,
 <tf.Tensor 'a/Softmax:0' shape=(？, 118) dtype=float32>,
 <tf.Tensor 'a/Softmax:0' shape=(？, 315) dtype=float32>]
```
Or the function will just work according to overall loss?
